src/main/webapp/jsp/instructorFeedbackResultsByGiverRecipientQuestion.jsp

```diff
@@ -470,9 +470,14 @@
                             <div class="col-md-10">
                     <%
                     	int qnIndx = 1;
+                        boolean isQnCONTRIB = false; /* is the Question a Contribution Question? */
                         for (FeedbackResponseAttributes singleResponse : responsesFromGiverToRecipient.getValue()) {
                             FeedbackQuestionAttributes question = questions.get(singleResponse.feedbackQuestionId);
                             FeedbackQuestionDetails questionDetails = question.getQuestionDetails();
+                            isQnCONTRIB = false;
+                            if(questionDetails.questionType == teammates.common.datatransfer.FeedbackQuestionType.CONTRIB) {                             
```

Comment:
You can avoid writing "teammates.common.datatransfer." by importing FeedbackQuestionType at the top of the jsp file.
Also, see if you can improve the indentation of the code added, as some of them looks off. You can ignore those lines not added by you.